flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java

```diff
@@ -47,16 +47,7 @@ public void actionPerformed(AnActionEvent e) {
       .addStep(new ChoseProjectTypeStep(model))
       .build();
     StudioWizardDialogBuilder builder = new StudioWizardDialogBuilder(wizard, "Create New Flutter Project");
-    // TODO(messick) Remove this reflection call after 3.2 is stable.
-    Method method = getMethod(builder.getClass(), "setUseNewUx", Boolean.class);
-    if (method != null) {
-      try {
-        method.invoke(builder, Boolean.TRUE);
-      }
-      catch (IllegalAccessException | InvocationTargetException e1) {
-        LOG.error(e1);
-      }
-    } // End of code to remove
+    builder.setUxStyle(StudioWizardDialogBuilder.UxStyle.DYNAMIC_APP);
```

Comment:
Can re remove the reflection imports in this file?

Reply:
done